mksfosinitrd.sh

```diff
@@ -73,6 +73,11 @@ check_files()
 	local FILES=$1
 	for f in $FILES; do
 		if test ! -e "$f"; then
+			# skip empty images folder
+			if [ "$f" = "res/images/*" ]; then
```

Comment:
Should we remove res/images from here? https://github.com/mer-hybris/hybris-initrd/blob/d20456ad5ffa53f434680e2830bd1a502cbb8549/mksfosinitrd.sh#L10

Reply:
If we remove it from TOOLS_LIST, it does not take those into initrd. I also tried to do symbolic link from /res/images to /usr/share/initrd-logos/ but that does not work either... That would have needed to modify hw-ramdisk and I don't want to touch it.